src/components/spark/spark.component.less

```diff
@@ -2,10 +2,12 @@ ux-spark {
     display: block;
 
     .ux-spark {
+        background: rgba(0, 167, 162, 0.2);
```

Comment:
Change to use `background-color` and use the `@primary` variable here too. To set the alpha value of a variable less has some useful function, I believe the `fade`/`fadeIn`/`fadeOut` functions will be the ones to look at